source/image/ImageSet.cpp

```diff
@@ -199,13 +124,11 @@ bool ImageSet::IsEmpty() const
 
 // Add a single image to this set. Assume the name of the image has already
 // been checked to make sure it belongs in this set.
-void ImageSet::Add(filesystem::path path)
+void ImageSet::Add(ImageFileData data)
```

Comment:
```suggestion
void ImageSet::Add(const ImageFileData &data)
```

Reply:
It seems that this would actually make an extra copy, because it's usually constructed in-place anyway, and I can't swap the path if it's const.